freqtrade/freqtradebot.py

```diff
@@ -129,6 +131,34 @@ def _throttle(self, func: Callable[..., Any], min_secs: float, *args, **kwargs)
         time.sleep(duration)
         return result
 
+    def refresh_tickers(self, pair_list: List[str]) -> Dict:
+        """
+        Refresh tickers asyncronously and return the result.
+        """
+        # TODO: maybe add since_ms to use async in the download-script?
+        # TODO: only refresh once per interval ? *may require this to move to freqtradebot.py
+        # TODO: Add tests for this and the async stuff above
+         
+        ticker_interval = self.strategy.ticker_interval
+        interval_in_seconds = int(ticker_interval[:-1]) * 60
```

Comment:
i wouldn't strip the last character, but instead use the conversation dict we have in Constants.py.
that way, we have a "default way" of doing it ...

Reply:
oops, nice one. going to change :)